backend/model/brandDbRequestModel.js

```diff
@@ -20,20 +20,16 @@ const brandSchema = new mongoose.Schema({
   },
   location: {
     type: String,
-    required: true,
+    default: "N/A",
```

Comment:
any specific reason why we are using `N/A` ?

Reply:
@Kajol-Kumari  In front-end we are not taking the location at signup, so I made it default `N/A`. later in the profile brand can update the location and other stuff.